gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -202,6 +202,17 @@ typedef struct _GstTensorFilterFramework
        *
        * @param[in] data the data element.
        */
+
+  int (*updateModel) (const GstTensorFilterProperties * prop,
+      void **private_data, const char * model_path);
```

Comment:
prop already has model info.

Reply:
Yes, I tried to reuse `PROP_MODEL` but it seems that its set_property is designed to handle model reopen. So, I wonder how we can distinguish between reopen vs. update_model